actions/admin/user/delete.php

```diff
@@ -27,7 +27,7 @@
 
 // forward to user administration if on a user's page as it no longer exists
 $forward = REFERER;
-if (strpos($_SERVER['HTTP_REFERER'], $username) != false) {
+if (strpos($_SERVER['HTTP_REFERER'], $username) > 0) {
```

Comment:
shouldn't it be `!== false`

Reply:
I think a better way would be to explode the path and check individual segments

Reply:
but what if it was in a query part? checking for presence is how it works now, let't not change it too much when trying to fix a syntax error